docs/man/sat-bootsys.8.rst

```diff
@@ -243,7 +243,7 @@ These options set the timeouts of various parts of the stages of the
 **--bos-shutdown-timeout** *BOS_SHUTDOWN_TIMEOUT*
         Timeout, in seconds, to wait until compute and
         application nodes have completed their BOS shutdown.
-        Defaults to 600. Overrides the option
+        Defaults to -1 for an infinite timeout. Overrides the option
```

Comment:
can we be specific for the user understanding. May be can mention like;
No default timeout is set. it is infinite. overrides the option